src/test/java/io/codiga/plugins/jetbrains/testutils/TestVariableMacroGeneric.java

```diff
@@ -0,0 +1,34 @@
+package io.codiga.plugins.jetbrains.testutils;
+
+import com.intellij.codeInsight.completion.CompletionType;
+import com.intellij.testFramework.PlatformTestUtil;
+import com.intellij.testFramework.fixtures.BasePlatformTestCase;
+import com.intellij.testFramework.fixtures.CodeInsightTestFixture;
+import com.intellij.testFramework.fixtures.IdeaTestExecutionPolicy;
+
+import java.util.List;
+
+public class TestVariableMacroGeneric extends BasePlatformTestCase {
```

Comment:
Add comment/documentation about what this class does. In two weeks, we will forget. Explain the goal and relationship with all other test classes.